config/redis.yml

```diff
@@ -2,9 +2,17 @@ development: &defaults
   redis:
     host: <%= Settings.redis.host %>
     port: <%= Settings.redis.port %>
+  # DO NOT CHANGE BELOW TTL (We have agreement with MHV on this for SSO)
   session_store:
     namespace: vets-api-session
-    each_ttl: 3600
+    each_ttl: 1800
+  user_b_store:
```

Comment:
not for this PR, but curious for details on the user_b_store naming

Reply:
in the past it was user, but at one point we needed to change it in a way that would otherwise not be reverse compatible, so we made something called user_b as a duplicate of user, eventually when we were confident that the data had trickled over properly we got rid of the duplicated logic and just switched to user_b. I figured we'd run into the same or similar issue again and we could change back to user when that time comes, but that time never came (so far).